apps/redi-connect/tailwind.config.js

```diff
@@ -1,77 +1,19 @@
 const { createGlobPatternsForDependencies } = require('@nrwl/react/tailwind')
 const { join } = require('path')
 
-// Configuration for TailwindCSS v3
 module.exports = {
   content: [
     join(__dirname, './src/**/*!(*.stories|*.spec).{js,ts,jsx,tsx,html}'),
+    join(
+      __dirname,
+      '../../libs/shared-shadcn-ui-components/src/**/*.{js,ts,jsx,tsx}'
+    ),
```

Comment:
Is using `shadcn` in the folder name a convetion from shadcn library? I'm not sure if it's really necessary to mention the name in the folder. Since we don't have `shared-ui-component`, how about simply use that instead?

Reply:
No, it's not a convention. We could use `shared-ui-components`, but we already have `shared-atomic-design-components`, which IMO would make it more confusing. Also, `shared-shadcn-ui-components` styled with TailwindCSS and `shared-atomic-design-components` - not.

Reply:
If the idea in the end is getting rid of all existing components (because of Bulma) and replacing them with Shadcn, I find it unnecessary to "trace" our components to the level of which library we used to generate them. This can be documented in the Readme so then everybody knows we are generating our ui components using Shadcn.

That being said, I don't really mind. I don't think it's worth a strong opinion, so we can definitely keep it like this.

Reply:
Agree with @katamatata. It can take months, even years, until bulma is removed, so until then...